postgres/upstream.sls

```diff
@@ -39,6 +39,6 @@ postgresql-repo:
 postgresql-repo:
   test.show_notification:
     - text: |
-        PostgreSQL does not provide package repository for {{ grains['osfinger'] }}
+        PostgreSQL does not provide package repository for {{ grains['os'] }}
```

Comment:
Does this work?

```suggestion
        PostgreSQL does not provide package repository for {{ grains['osfinger'] | d(grains['os']) }}
```

That gives us the best of both worlds.

Reply:
It's a jinja rendering error so that will not fix.
 If we really want osfinger then will need a if/else block

Reply:
I think calling `salt['grains.get']('osfinger', grains['os'])` would be a better choice here. No need for additional conditional statements.
Since that is just a notification message, having exactly `osfinger` value substituted is not such critical matter.

Reply:
Thanks @myii and @vutny for feedback.  I have updated as suggested by @vutny and since `salt['grains.get']` swallows the exception this should work in all cases.